polaris-tokens/src/themes/base/color.ts

```diff
@@ -400,1106 +235,631 @@ export const color: {
-  },
-  'color-text-interactive': {
-    value: colorsExperimental.blue[13],
-  },
-  'color-text-interactive-inverse': {
-    value: colorsExperimental.blue[8],
-  },
-  'color-text-magic-strong': {
-    value: colorsExperimental.purple[15],
+    value: colors.azure[14],
   },
   'color-text-magic': {
-    value: colorsExperimental.purple[14],
+    value: colors.purple[14],
   },
   // ------------------------------
   // Net new tokens
   // ------------------------------
   'color-border-inverse-active': {
-    value: colorsExperimental.gray[8],
+    value: colors.gray[8],
   },
   'color-border-inverse-hover': {
-    value: colorsExperimental.gray[10],
+    value: colors.gray[10],
   },
   'color-border-warning': {
-    value: colorsExperimental.orange[8],
+    value: colors.orange[8],
   },
   'color-checkbox-icon-disabled': {
-    value: colorsExperimental.gray[1],
+    value: colors.gray[1],
   },
   'color-checkbox-bg-surface-disabled': {
-    value: colorsExperimental.blackAlpha[7],
+    value: colors.blackAlpha[7],
   },
   'color-bg-fill-active': {
-    value: colorsExperimental.gray[4],
+    value: colors.gray[4],
   },
   'color-bg-fill-brand-selected': {
-    value: colorsExperimental.gray[15],
+    value: colors.gray[15],
   },
   'color-bg-fill-caution-active': {
-    value: colorsExperimental.yellow[9],
+    value: colors.yellow[9],
   },
   'color-bg-fill-caution-hover': {
-    value: colorsExperimental.yellow[8],
+    value: colors.yellow[8],
   },
   'color-bg-fill-critical-selected': {
-    value: colorsExperimental.red[14],
+    value: colors.red[14],
   },
   'color-bg-fill-emphasis-active': {
-    value: colorsExperimental.blue[15],
+    value: colors.blue[15],
   },
   'color-bg-fill-emphasis-hover': {
-    value: colorsExperimental.blue[14],
+    value: colors.blue[14],
   },
   'color-bg-fill-emphasis': {
-    value: colorsExperimental.blue[13],
+    value: colors.blue[13],
   },
   'color-bg-fill-hover': {
-    value: colorsExperimental.gray[3],
+    value: colors.gray[3],
   },
   'color-bg-fill-info-active': {
-    value: colorsExperimental.azure[11],
+    value: colors.azure[11],
   },
   'color-bg-fill-info-hover': {
-    value: colorsExperimental.azure[10],
+    value: colors.azure[10],
   },
   'color-bg-fill-secondary-active': {
-    value: colorsExperimental.gray[8],
+    value: colors.gray[8],
   },
   'color-bg-fill-secondary-hover': {
-    value: colorsExperimental.gray[7],
+    value: colors.gray[7],
   },
   'color-bg-fill-selected': {
-    value: colorsExperimental.gray[10],
+    value: colors.gray[10],
   },
   'color-bg-fill-transparent-secondary-active': {
-    value: colorsExperimental.blackAlpha[8],
+    value: colors.blackAlpha[8],
   },
   'color-bg-fill-transparent-secondary-hover': {
-    value: colorsExperimental.blackAlpha[7],
+    value: colors.blackAlpha[7],
   },
   'color-bg-fill-transparent-selected': {
-    value: colorsExperimental.blackAlpha[6],
+    value: colors.blackAlpha[6],
```

Comment:
```suggestion
    value: colors.blackAlpha[7],
```
This is listed as [7 in Figma](https://www.figma.com/file/oKq2CLPYeX5qK5frzwipM3/Polaris-Gen-3-Styles?type=design&node-id=14696%3A1737&mode=design&t=vGyfRzAa3mogdM3j-1). We need to determine which of these values is correct.

<img width="510" alt="Screenshot 2023-10-02 at 12 09 02 PM" src="https://github.com/Shopify/polaris/assets/21976492/ebc3d966-ebed-4199-9547-75c2b9c6ed37">